package.json

```diff
@@ -20,12 +16,17 @@
     "grunt-shell": "1.3.0",
     "shelljs": "0.3.0"
   },
+  "scripts": {
+    "start": "electron main.js",
+    "test": "electron tests.js",
+    "postinstall": "cd ./node_modules/leveldown && node-gyp rebuild --target=1.4.1 --dist-url=https://atom.io/download/atom-shell"
```

Comment:
The secret was 'vagrant provision' thanks @kaspermarkus! I suppose the different version of Node was causing the error I was seeing.